dvc/tree/azure.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, repo, config):
 
         self._account_url = None
         if not self._conn_str:
-            name = self._az_config.get("storage", "account", None)
+            name = config.get("account")
```

Comment:
```suggestion
            name = config.get("account") or self._az_config.get("storage", "account", None)
```

Reply:
Sorry; let's revert it back. This code still doesn't work.